spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/TestSparkWriteConf.java

```diff
@@ -486,6 +486,36 @@ public void testDeleteFileWriteConf() {
     }
   }
 
+  @Test
+  public void testExtraSnapshotMetadataReflectsSessionConfig() {
+    withSQLConf(
+        ImmutableMap.of("spark.sql.iceberg.snapshot-property.test-key", "test-value"),
```

Comment:
Trivial nit: "test-value" -> "session-value" so it mimics the test below. This is the tiniest of nits so feel free to ignore.